enemies/bosses/main.json

```diff
@@ -420,6 +428,7 @@
         ]
       },
       "hp": 13500,
+      "expectedDamageFrames": 1200,
```

Comment:
Why would this fight take more damage if it lasts longer? You don't move and he can't hit you.

Reply:
My train of thought was something along the line of taking damage from thrown back supers that sometimes occur. Would you like to use 0?

Reply:
I think the trajectory of those Supers may depend on your position, but yeah that's unclear.